addons/controllers/csi/vspherecsiconfig_utils.go

```diff
@@ -360,3 +367,26 @@ func (r *VSphereCSIConfigReconciler) getNumberOfControlPlaneNodes(ctx context.Co
 	}
 	return *kcp.Spec.Replicas, nil
 }
+
+// Return true if Valid Availability Zones are present
+// i.e. if the cluster is Stretch Supervisor
+func (r *VSphereCSIConfigReconciler) isStretchedSupervisorCluster(ctx context.Context) (bool, error) {
+	azList := &topologyv1alpha1.AvailabilityZoneList{}
+
+	err := r.Client.List(ctx, azList)
+	if err != nil {
+		return false, err
+	}
+
+	if len(azList.Items) <= 0 {
+		return false, nil
+	}
+
+	if len(azList.Items) == 1 {
```

Comment:
can you have more than 1 AZ?

Reply:
no

Reply:
ok. please add a comment explaining that when "LegacyZoneName" is in the list, it is the only item in the list.